client/client.go

```diff
@@ -55,11 +55,11 @@ func (l logger) Infof(s string, args ...interface{}) {
 }
 
 func (l logger) Warningf(s string, args ...interface{}) {
-	l.Infof(s, args)
+	l.Infof(s, args...)
```

Comment:
Why has this changed?

Reply:
It seems that we moved to the latest release of the linter and that is catching more issues. Let me fix the linter to this version and leave it there.

`export GOLANGCILINT=$(curl -fsSLI -o /dev/null -w %{url_effective} https://github.com/golangci/golangci-lint/releases/latest | awk -F '/' '{print $8}')`